lib/ios/UITabBar+centered.m

```diff
@@ -0,0 +1,53 @@
+#import "UITabBar+centered.h"
+#import <objc/runtime.h>
+
+#define BADGE_OFFSET 0.2
+#define IMAGE_VIEW_TAG 1
+
+typedef void (*UITabBarButton_layoutSubviews__IMP)(void);
+static UITabBarButton_layoutSubviews__IMP original_UITabBarButton_layoutSubviews;
+
+@implementation UITabBar (centered)
+
+- (void)centerTabItems {
+	static dispatch_once_t onceToken;
+	dispatch_once(&onceToken, ^{
+		[[self class] rnn_extendsUITabBarButtonLayoutSubviews];
+	});
+	
+	for (UITabBarItem* item in self.items) {
+		item.title = nil;
+	}
+}
+
+- (void)rnn_UITabBarButton_layoutSubviews {
```

Comment:
As this function is only used from the function below - moving it below it will improve readability as we'd be able to read this file top to bottom. Right now we start at the top, continue to the third function then finish at the middle of the file in the second function.